h/schemas/annotation.py

```diff
@@ -311,3 +311,86 @@ def _target_selectors(targets):
         return targets[0]['selector']
     else:
         return []
+
+
+class SearchParamsSchema(JSONSchema):
+    """Validates a JSON payload for an elasticsearch query via API"""
+    schema = {
+        # 'type': 'object',
+        'properties': {
+            '_separate_replies': {
+                'type': 'string',
+            },
+            'sort': {
+                'type': 'string',
+            },
+            'limit': {
+                'type': 'string',
+            },
+            'order': {
+                'type': 'string',
+                'enum': ["asc", "desc"],
+            },
+            'offset': {
+                'type': 'string',
+            },
+            'created': {
+                'type': 'string',
+            },
+            'group': {
+                'type': 'string',
+            },
+            'quote': {
+                'type': 'string',
+            },
+            'references': {
+                'type': 'string',
+            },
+            'tag': {
+                'type': 'string',
+            },
+            'tags': {
+                'type': 'string',
+            },
+            'text': {
+                'type': 'string',
+            },
+            'target': {
+                'type': 'string',
+            },
+            'thread_ids': {
+                'type': 'string',
+            },
+            'updated': {
+                'type': 'string',
+            },
+            'uri': {
+                'type': 'string',
+            },
+            'url': {
+                'type': 'string',
+            },
+            'any': {
+                'type': 'string',
+            },
+            'user': {
+                'type': 'string',
+            },
+        },
+        'required': [],
+    }
+
+    def validate(self, data):
```

Comment:
I didn't make a different validator but rather overrode the validate method. This seemed like the simplest approach but I'm open to other ideas. The validator we are using already ignores the unknown properties but it passes them through rather than removing them so all we have to do after the validator runs is remove them.